react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+class Title extends React.Component {
+  render() {
+    const title = this.props.title;
+    const id = this.props.id;
+    return (
+      <tr className="title">
+        <td id={id}>{title}</td>
+      </tr>
+    );
+  }
+}
+
+class Content extends React.Component {
+  render() {
+    const content = this.props.content;
+    return (
+        <tr>
+          <td>{content}</td>
+        </tr>
+    );
+  }
+}
```

Comment:
If you aren't using `state` or a lifecycle method it is best to use a `function` component.